virttest/utils_logfile.py

```diff
@@ -30,7 +30,7 @@
 _open_log_files = {}  # pylint: disable=C0103
 
 
-def _acquire_lock(lock, timeout=10):
+def _acquire_lock(lock, timeout=30):
```

Comment:
Hi @pevogam,

I agree that this is a solid temporary workaround for performance issues on constrained hardware.

One thought: should we parameterize the timeout? If we update `log_line` to take a `timeout` argument (defaulting to 10) and pass it to `_acquire_lock`, it would make the system more flexible. That way, users on very slow hardware can increase it themselves, and we won't have to keep raising the default hard limit (e.g., to 30 or higher).

In any case, the current solution works for me. Thanks!